### ISSUE TITLE
Hero carousel not working correctly on desktop

### ISSUE THREAD

**Issue by kronnox** [alexsaar/anymotion-dance#23]
- After clicking on the slide navigation to change to another slide the indicator isn't updated correctly (stuck on the previous page)
- Maybe a bug / Maybe an improvement: Feature parity with mobile; click and drag to the side to change slides

**Comment by kronnox** [comment]
Bulletpoint 1 seems to vary depending on the screen size. In smaller windows its working in bigger ones it isn't.

**Comment by kronnox** [comment]
> Feature parity with mobile; click and drag to the side to change slides

@alexsaar Is that something we want? Should be done with a bit of simple JS, but I don't know if it's even something that would improve the UX much.

**Comment by alexsaar** [comment]
I cannot reproduce the first bullet. works well on full screen with 27 inch display. tested with FF, Safari and Chrome. there is a bit of a delay though updating the nav. what setup are you using?

regarding second bullet: my feel it is consistent across mobile and desktop, in both cases it works with scrolling rather than d&d. Or did you mean something different?

**Comment by kronnox** [comment]
> I cannot reproduce the first bullet. works well on full screen with 27 inch display. tested with FF, Safari and Chrome. there is a bit of a delay though updating the nav. what setup are you using?

@alexsaar After taking a look at the code, I was able to find the cause (and circumstances to reproduce) for this behaviour. It will occur on any monitor where the carousel is filling up the whole screen. This is also the case on any regular Full-HD 16:9 monitor (see attached image).

One solution would be to change the way the current slide is detected. Another option would be to change the responsive design in a way that the carousel never fills the whole screen (in my opinion it shouldn't anyways, from a design standpoint). Or maybe both.

<img width="400" alt="Screenshot 2023-04-17 at 09 16 34" src="https://user-images.githubusercontent.com/28481491/232413148-6f9bcb20-6d0b-4a4a-96b5-29dd44756d91.png">

**Comment by kronnox** [comment]
> regarding second bullet: my feel it is consistent across mobile and desktop, in both cases it works with scrolling rather than d&d. Or did you mean something different?

As far as I can see, the only way to change slides on desktop is by clicking the bullet points on the bottom. I was contemplating whether we should add the possibility to change slides by "swiping" similar to how it would be with touch or add buttons on the left and right. The more I think about it the less I like this approach; I would leave it as is. Just wanted to bring it up.

**Comment by alexsaar** [comment]
> One solution would be to change the way the current slide is detected. Another option would be to change the responsive design in a way that the carousel never fills the whole screen (in my opinion it shouldn't anyways, from a design standpoint). Or maybe both.

@Kronnox I agree, the design and responsive layout should be improved. I tuned it a bit, but it's not super great just yet. Also with the existing images, on some screens it cuts off the main subject.

and yes, while we are at it we can also improve the "current slide" detection part.

**Comment by alexsaar** [comment]
> > regarding second bullet: my feel it is consistent across mobile and desktop, in both cases it works with scrolling rather than d&d. Or did you mean something different?
> 
> As far as I can see, the only way to change slides on desktop is by clicking the bullet points on the bottom. I was contemplating whether we should add the possibility to change slides by "swiping" similar to how it would be with touch or add buttons on the left and right. 

that works for me. using a track pad admittedly, I can easily swipe (2 finger) and with this scroll through the hero banners.
